### PR TITLE
Work for review

### PR DESCRIPTION
**Pull request by Kwolik**
image

1. Create page: WorkForReviewComponent.jsx, work-for-review.js, CurrentVersion.jsx

### CODE REVIEW

**Review by upsu90 — CHANGES_REQUESTED** [review]
Failed to compile :c 
image

File: web-client/src/pages/work-for-review.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  function handleOnClick(item) {
+    for (let i = 0; i < open.length; i++) {
+      // open clicked version
+      if (item === i && !open[i]) 
+        open[i] = true;
+      // otherwise close it
+      else 
+        open[i] = false;
+    }
+
```
Comment (upsu90) [inline web-client/src/pages/work-for-review.js]: I think it'll be better to be able to open every version at the same time

File: web-client/src/components/WorkForReviewComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+                        </DialogActions>
+                    </form>
+                </Dialog>
+            </div>
+                {/*DropDown*/}
+                {/* <Select options={'lol'}/> */}
+                {/* <Dropdown.Menu show>
+                    <Dropdown.Item>Siema</Dropdown.Item>
+                    <Dropdown.Item>eniu</Dropdown.Item>
+                </Dropdown.Menu> */}
```
Comment (upsu90) [inline web-client/src/components/WorkForReviewComponent.jsx]: ;)

File: web-client/src/components/WorkForReviewComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+                        </DialogContent>
+                        <DialogContent className={style.drag}>
+                            <DropZone SET_FILE={passFile}
+                                inputRef={register}
+                                required
+                            />
+                        </DialogContent>
+                        <DialogActions>
+                            <Button variant='contained' color="primary"
+                                className={style.btn1} type='submit'>Add review</Button>
```
Comment (upsu90) [inline web-client/src/components/WorkForReviewComponent.jsx]: align it, please
image

**Review by upsu90 — CHANGES_REQUESTED** [review]

File: web-client/src/components/WorkForReviewComponent.jsx (hunk 2 above)
Comment (Kwolik) [reply web-client/src/components/WorkForReviewComponent.jsx]: :)

Comment (JustD4nTe) [reply web-client/src/components/WorkForReviewComponent.jsx]: :)

**Pull request closed without merging**